common/thorhelper/thorsoapcall.cpp

```diff
@@ -989,6 +989,7 @@ class CWSCHelper : implements IWSCHelper, public CInterface
         : logctx(_logctx), outputAllocator(_outputAllocator), clientCert(_clientCert), roxieAbortMonitor(_roxieAbortMonitor)
     {
         activitySpanScope.setown(logctx.queryActiveSpan()->createInternalSpan(_wscType == STsoap ? "SoapCall Activity": "HTTPCall Activity"));
+        activitySpanScope->setSpanAttribute("activity_id", _rowProvider->queryActivityId());
```

Comment:
All other attributes seem to use dot notation

Reply:
They use dot notation for the scope, but underscore within a scope.  E.g.
`http.response.status_code`

Reply:
I started off with hpcc.activity_id, but then removed the hpcc. prefix for the same reason that we removed it from the stat names.

Reply:
activity_id is fine, just pointing out all other attributes use dots rather than underscores. I was thinking activity.id
that implies there could be multiple activity.* attributes.